### DISCUSSION TITLE
Scripts failing

### DISCUSSION THREAD

**Discussion by adyfoot** [tteck/Proxmox#2140] [Help Needed] [answered]
Hi folks - seem to be having trouble this evening.  Trying to setup some LXCs, but keep getting this issue:

```
Creating a Docker LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using SSD_Data for Container Storage.
 - Updating LXC Template List...
update failed - see /var/log/pveam.log for details
 ✓ Updated LXC Template List
 - Downloading LXC Template...download failed: Connecting to download.proxmox.com (download.proxmox.com)|2407:6180:1:a:70dc:f0ff:fe97:4cbb|:80... failed: Network is unreachable.
 ✓ Downloaded LXC Template
```
Any help would be appreciated.  DNS is setup properly and the Proxmox server is connected to the internet properly.

**Comment by tteck** [comment]
Disable IPv6 in your router.

**Reply by adyfoot** [reply]
I'm not using IPv6...

**Reply by tteck** [reply]
> (download.proxmox.com)|2407:6180:1:a:70dc:f0ff:fe97:4cbb|:80... failed: Network is unreachable.

Really?

**Reply by adyfoot** [reply]
Yes, really. This was working previously BTW. This is just a fresh Proxmox install and needed to get things set back up again. Perhaps my local Proxmox CDN is down?

**Reply by adyfoot** [reply, marked as answer]
Woke up this morning and tried this again - with no changes anywhere on Proxmox or my network - and all is now working.  So I think my conclusion of the Proxmox CDN being down was correct.  Thanks for just closing the ticket without any further discussion though...
